### PR TITLE
Implement Logos block. Define fragments in build

### PR DESCRIPTION
**Pull request by ozasadnyy**
Phone view:

Desktop view:

### CODE REVIEW

File: data/resources.json [hunk 1]
```diff
@@ -117,5 +118,9 @@
     "title": "Tickets",
     "ticketsDetails": "Tickets grant access to all conference sections, coffee breaks, lunch and party. Accommodation is NOT included in the ticket price.",
     "soldOut": "Sold out"
+  },
+  "logosBlock": {
```
Comment (sophieH29) [inline data/resources.json]: Rename to partners block please

Comment (ozasadnyy) [inline data/resources.json]: no, as people will be able to use elsewhere. For instance: "Customers", etc.

File: src/elements/logos-block.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
```
Comment (sophieH29) [inline src/elements/logos-block.html]: I think the appropriate name is PartnersBlock. LogosBlock is and old version, and always seemed to me weird

Comment (ozasadnyy) [inline src/elements/logos-block.html]: See comment above

File: src/elements/logos-block.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        .logo-item {
+          width: calc(25% - 16px);
+        }
+      }
+
+      @media (min-width: 812px) {
+        .logo-item {
+          width: calc(20% - 16px);
+        }
+      }
```
Comment (sophieH29) [inline src/elements/logos-block.html]: you can leave just @media (min-width: 640px)

File: src/elements/logos-block.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      :host {
+        display: block;
+      }
+
+      .block-title {
+        margin: 24px 0 8px;
+      }
+
+      .logos {
+        margin: 0 -8px;
```
Comment (sophieH29) [inline src/elements/logos-block.html]: -8 ? don't you have enough height?

Comment (ozasadnyy) [inline src/elements/logos-block.html]: It affect container width, not height

File: src/elements/logos-block.html (hunk 3 above)
Comment (ozasadnyy) [inline src/elements/logos-block.html]: > 640px - 4 blocks in a row
> 812px - 5 blocks

**Pull request merged**